distribution/bin/log4j2.xml

```diff
@@ -67,5 +67,9 @@
         <Logger name="org.eclipse.persistence" level="info" additivity="false">
             <AppenderRef ref="Routing"/>
         </Logger>
+        <Logger name="org.wso2.testgrid.automation.executor.JMeterResultCollector" additivity="false">
+            <AppenderRef ref="Routing" level="info"/>
+            <AppenderRef ref="TESTGRID_CONSOLE" level="error" />
+        </Logger>
```

Comment:
Logging into Routing and TESTGRID_CONSOLE is the default behavior (as defined by 'Root' above). Hence, we don't need this.

Reply:
Fixed by introducing a separate logger for appending jmeter test results in 50ccef6